Added ButaneDAO

bbcchain opened the pull request:
#### Hi! What is your PR about?

- [ ] Add or edit a skin
- [x] Add a custom domain for your space
- [x] Add an alias to migrate your space

gokaiorg commented on spaces/aliases.json:
```diff
@@ -62,6 +62,7 @@
   "tptdao.eth": "itokenpocket.eth",
   "tallyhodao.eth": "tahodao.eth",
   "goodghosting.eth": "halofi.eth",
+  "butanedao.eth": "butanedao.eth",
```
Use aliases.json when you aim to proceed to a space migration.
```suggestion
```

bbcchain replied:
> Use aliases.json when you aim to proceed to a space migration.

I have updated the PR

gokaiorg commented on spaces/aliases.json:
```diff
@@ -62,6 +62,7 @@
   "tptdao.eth": "itokenpocket.eth",
   "tallyhodao.eth": "tahodao.eth",
   "goodghosting.eth": "halofi.eth",
+  "vote.butanedao.io": "butanedao.eth",
```
You don't need to edit the aliases file, it is reserved for space migrations. 
```suggestion
```

bbcchain reviewed (commented):
I hope this is correct

gokaiorg commented:
> I hope this is correct

Nope 😁 By editing the aliases file you mean you want to migrate https://snapshot.org/#/butanedao.io to https://snapshot.org/#/butanedao.eth. But the butenedao.io space does not exist.

bbcchain commented:
I deleted the space thinking I could redo it. Im trying to get vote.butanedao.io to work. We have also added our chain in chain list. Could you please make necessary changes?

Thank you

gokaiorg commented:
I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.org/#/butanedao.eth

gokaiorg commented:
Closing this PR for now, feel free to open it again once you've done the changes.
